test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/Resources/TestAWSLambdaResourceDetector.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="TestAWSLambdaResourceDetector.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Linq;
+using OpenTelemetry.Contrib.Extensions.AWSXRay.Resources;
+using Xunit;
+
+namespace OpenTelemetry.Contrib.Extensions.AWSXRay.Tests.Resources
+{
+    public class TestAWSLambdaResourceDetector
+    {
+        [Fact]
+        public void TestDetect()
+        {
+            Environment.SetEnvironmentVariable("AWS_REGION", "us-east-1");
+            Environment.SetEnvironmentVariable("AWS_LAMBDA_FUNCTION_NAME", "testfunction");
+            Environment.SetEnvironmentVariable("AWS_LAMBDA_FUNCTION_VERSION", "latest");
```

Comment:
Just as a good practice, you should clean up any env var that you set in your tests.
It may not happen here but can lead to some weird behavior if some other tests depended on these env vars in the future.

Reply:
Thanks, just updated